README.md

```diff
@@ -86,11 +87,13 @@ environment variable
 
 ##### Steps for Windows platform
 
-* Go to directory with cloned repository
+* Go to the directory with the cloned repository and enter the subfolder tools\ms.
 * Run setup_VS2013_x64.cmd to generate solution for Visual Studio 2013.
 **Note:** In case Visual Studio has been installed to non-default location -- modify path in .cmd file.
 **Note:** Double click on file or run from windows command prompt. DO NOT run in "git bash".
-* After script execution finish, go to build directory which is located one directory upper and open smartDeviceLinkCore.sln file using Visual Studio 2013
+* After script execution finish:
+  * Add path to the openssl executable to the PATH environment variable. Usually in build\openssl_win_x64\bin.
+  * Go to the build directory which is located one directory upper and open smartDeviceLinkCore.sln file using Visual Studio 2013
```

Comment:
@anosach-luxoft fixed in [07ed25a](https://github.com/smartdevicelink/sdl_core/pull/791/commits/07ed25aea7676d504be507c2c11b640656576e3e)